docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -37,128 +37,121 @@ To find the `volumeHandle`, use one of the following procedures:
-      # persistentVolumeReclaimPolicy: Retain
-      storageClassName: ibmc-block-gold
-      # volumeMode: Filesystem
-    </pre>
-
-3.  Create a PersistentVolumeClaim (PVC) yaml file.
+2. Take the `volume_name` and other optional information (collected before the procedure) and insert it into the YAML file (under `spec.csi.volumeAttributes`).
+
+    **Important:** If using the CSI Topology feature, the `spec.csi.volumeHandle` contains the system ID. In the example below, the `spec.csi.volumeHandle` would read similar to the following: `SVC:demo-system-id-1:600507640082000B08000000000004FF`.
+    
+        apiVersion: v1
+        kind: PersistentVolume
+        metadata:
+          # annotations:
+            # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
+          name: vol1-pv
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          capacity:
+            storage: 1Gi
+          csi:
+            controllerPublishSecretRef:
+              name: demo-secret
+              namespace: default
+            driver: block.csi.ibm.com
+            # volumeAttributes:
+              # pool_name: ibmc-block-gold
+              # storage_type: SVC
+              # volume_name: vol1
+              # array_address: baremetal10-cluster.xiv.ibm.com
+            volumeHandle: SVC:600507640082000B08000000000004FF
+          # persistentVolumeReclaimPolicy: Retain
+          storageClassName: ibmc-block-gold
+          # volumeMode: Filesystem
+
+3. Create a PersistentVolumeClaim (PVC) YAML file.
 
     **Note:**
 
-    -   To include a specific 5 Gi PV, be sure to include the `storageClassName`.
-    -   For more information about creating a PVC yaml file, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+    - To include a specific 5 Gi PV, be sure to include the `storageClassName`.
+    - For more information about creating a PVC YAML file, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
     
-    ```screen
-    apiVersion: v1
-    kind: PersistentVolumeClaim
-    metadata:
-      # annotations:
-        # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
-      name: vol1-pvc
-    spec:
-      accessModes:
-      - ReadWriteOnce
-      resources:
-        requests:
-          storage: 1Gi
-      storageClassName: ibmc-block-gold
-      volumeName: vol1-pv
-    ```
-
-4.  Create a StatefulSet.
-
-    For more information about creating a StatefulSet, see [Creating a StatefulSet](csi_ug_config_create_statefulset.md).
-
-    ```screen
-    kind: StatefulSet
-    apiVersion: apps/v1
-    metadata:
-      name: sanity-statefulset
-    spec:
-      selector:
-        matchLabels:
-          app: sanity-statefulset
-      serviceName: sanity-statefulset
-      replicas: 1
-      template:
+            apiVersion: v1
+            kind: PersistentVolumeClaim
+            metadata:
+              # annotations:
+                # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
```

Comment:
I don't think this is needed for PVC

Reply:
@tyichye19  you reviewed this for last release. should I keep or remove? Thanks

Reply:
@tyichye19 was this needed for the PVC?

Reply:
I think we can remove those lines.
last time I used the import option it was without those lines and it worked fine.

Reply:
@tyichye19  @oriyarde  remove it from each instance in the file? PVC and PV yamls?

Reply:
PVC for sure, not sure about PV

Reply:
DONE. let's just leave it for PV for now. it's too soon before GA and it's marked specifically as not optional so it really shouldn't make a difference.